packages/@aws-cdk/aws-cloudtrail/test/test.cloudtrail.ts

```diff
@@ -207,6 +210,46 @@ export = {
         test.deepEqual(trail.DependsOn, [logsRolePolicyName, logsRoleName, 'MyAmazingCloudTrailS3Policy39C120B0']);
         test.done();
       },
+
+      'enabled and with custom log group'(test: Test) {
```

Comment:
minor: i quite like the // GIVEN // WHEN //THEN when we do it.
we aren't super consistent about it so do with this comment what you will.

Reply:
I'm ambivalent on this. Doesn't exist on the rest of this file. I'm leaving this as is, in favour of consistency.